src/commands/init.js

```diff
@@ -46,13 +46,23 @@ exports.handler = function ({ reporter, name, template }) {
   
   const basename = name.split('.')[0]
   const tasks = new TaskList([
+    {
+      title: 'Checking project existence',
+      task: async (ctx, task) => {
+        const projectPath = path.resolve(process.cwd(), basename)
+        const exists = await fs.pathExists(projectPath)
+        if (exists) {
+          throw new Error(`Couldn't initialize project. Project with name ${basename} already exists in ${projectPath}. Use different <name> or rename existing project folder.`)
+        }
+      }
+    },
     {
       title: 'Clone template',
       task: async (ctx, task) => {
         task.output = `Cloning ${template} into ${basename}...`
 
         const repo = await clone(template, basename, { shallow: true })
-        console.log(`Template cloned to ${basename}`)
+        // console.log(`Template cloned to ${basename}`)
```

Comment:
How would you feel about removing it entirely instead of leaving a comment here @izqui ? :)

Also, would you mind enlighten me how having this console.log causes multiple outputs of a totally different task?

Reply:
Your are absolutely right, forgot to remove it after testing. Fixed: https://github.com/aragon/aragon-cli/pull/165

`TaskList` basically hijacks the console output and if you try to print in a different way, the output gets fucked up.

Reply:
> TaskList basically hijacks the console output and if you try to print in a different way, the output gets fucked up.

Wow.. I had no idea. Thanks!